src/libasr/codegen/asr_to_llvm.cpp

```diff
@@ -815,6 +815,54 @@ class ASRToLLVMVisitor : public ASR::BaseVisitor<ASRToLLVMVisitor>
         return builder->CreateCall(fn, {str, idx1, idx2});
     }
 
+    llvm::Value* lfortran_float_to_str(llvm::Value* number)
+    {
+        std::string func_name = "_lfortran_float_to_str";
+        llvm::Function *fn = module->getFunction(func_name);
+        if (!fn) {
+            llvm::FunctionType *function_type = llvm::FunctionType::get(
+                    character_type, {
+                        llvm::Type::getFloatTy(context)
+                    }, false);
+            fn = llvm::Function::Create(function_type,
+                    llvm::Function::ExternalLinkage, func_name, *module);
+        }
+        llvm::Value* res = builder->CreateCall(fn, {number});
+        return res;
+    }
+
+    llvm::Value* lfortran_int_to_str(llvm::Value* number)
+    {
+        std::string func_name = "_lfortran_int_to_str";
+        llvm::Function *fn = module->getFunction(func_name);
+        if(!fn) {
+            llvm::FunctionType *function_type = llvm::FunctionType::get(
+                character_type, {
+                    llvm::Type::getInt32Ty(context)
```

Comment:
Use, `getIntType` defined in `asr_to_llvm.cpp`. By the way what if I pass a 64 bit integer as input to `str`. Will this code still work?